geometry_msgs/Point point_local
geometry_msgs/Point point_global
geometry_msgs/Vector3 min_local
geometry_msgs/Vector3 max_local
geometry_msgs/Vector3 min_global
geometry_msgs/Vector3 max_global
int64 cluster_size
bool normal_is_inward
int64 correspond_lm_id
